### PR TITLE
Update custom-prom.md

### PR DESCRIPTION
**Pull request by praveenkumar0566**
Added Data Retention step

### CODE REVIEW

File: custom-prom.md [hunk 1]
```diff
@@ -222,3 +222,8 @@ Good:
 In Kubecost, you can view basic diagnostic information for Prometheus metrics by selecting _Settings_ in the left navigation, then scrolling down to Prometheus Status, as seen below:
 
 ![Prometheus status diagnostic](./images/prom-status.png)
+
+### Data Retention
+By default metric retention is 91 days, Howerver the retenion of data can be further increased with a configurable value for a property **etlDailyStoreDurationDays**
```
Comment (srpomeroy) [inline custom-prom.md:227]:
```suggestion
By default, metric retention is 91 days, however the retention of data can be further increased with a configurable value for a property **etlDailyStoreDurationDays**
```

**Review by srpomeroy — CHANGES_REQUESTED** [review]

**Comment by praveenkumar0566** [comment]
Hi @srpomeroy Is there anything i need to address to enable merge? I have added the review comments provided.

**Review by bstuder99 — APPROVED** [review]

**Comment by bstuder99** [comment]
Sorry, was watching over this last week but wasn't sure if it had been resolved. I can give out a review

**Comment by thomasvn** [comment]
Thanks @praveenkumar0566 for getting started on this! This is a pretty important config.

@bstuder99 This config allows users to configure the amount of days Kubecost will serve via API and Frontend. The default is 90 days. Because this is a frequently asked about config, do you think it should eventually become a stand-alone doc under the "Advanced Configuration" section? We should also reference this config in the "Cost Allocation" and "Assets" doc.

**Pull request merged**